packages/expo-notifications/README.md

```diff
@@ -1,5 +1,115 @@
 # expo-notifications
 
-`expo-notifications` provides some great stuff.
+## Instalation 
+
+`expo-notifications` needs `react-native-unimodule` to work. You can read more about it [here](https://www.npmjs.com/package/react-native-unimodules). 
+
+Add `expo-notifications` to your project:
+execute the following command in your project root directory:
+"yarn add expo-notifications".
+
+### Android
+
+#### 1. Modify AndroidManifest
+
+`AndroidManifest.xml` is a place where you can configure `expo-notifications` module. For Instance, you can choose there
+if you want to use Expo's servers as a middleman or choose that you don't want to use push notifications at all.
+
+Configuration template:
+
+```xml
+
+<!-- expo-notifications START -->
+    <!-- turn on push notifications START -->
+        <service
+        android:name="expo.modules.notifications.push.fcm.ExpoFcmMessagingService"
+        android:exported="false">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service> 
+
+        <!-- choose if you want to use expo servers START -->
+        <meta-data android:name="appId" android:value="@username/slug" /> <!-- only with expo engine -->
```

Comment:
Shouldn't we use some more qualified names for the `meta-data` so that they never conflict with other values (like eg. Google Mobile Ads App ID's key is `com.google.android.gms.ads.APPLICATION_ID`, not `googleMobileAdsAppId` or worse `appId`)?